amr-wind/equation_systems/vof/vof_ops.H

```diff
@@ -32,7 +32,7 @@ struct FieldRegOp<VOF, Scheme>
         auto& iomgr = sim.io_manager();
         iomgr.register_io_var(fields.field.name());
         iomgr.register_io_var(levelset.name());
```

Comment:
Are these auxiliary variables required for restart? If not, I would recommend using `iomgr.register_output_var` for these aux variables.

Reply:
Done